### DISCUSSION TITLE
Justified text

### DISCUSSION THREAD

**Discussion by HS-MJ** [Seniblue/Underwater#12] [Ideas]
Hey, still loving this theme, especially the warm tones of the biscuit colour scheme. May I request that the justified text is made optional in Style Settings? I have it commented out in the theme CSS as I prefer left aligned, but would be nice to be less manual 😄

**Comment by Seniblue** [comment]
I'll add a setting for this in the next version (towards the end of the week). Thanks for the feedback!

**Comment by Seniblue** [comment]
Version 1.5.3 is out :)
You can disable justified text in "🔧 Options" (via Style settings)
